### ISSUE TITLE
Optional Apps Pane after inital policy_array

### ISSUE THREAD

**Issue by GadgetGeekNI** [dan-snelson/dialog-scripts#37]
> **Note:** While all requests are welcome, finding available cycles to custom-code a feature we won't use in our production environment is always challenging.

**Is your feature request related to a problem? Please describe.**
This is not a problem and only a nice to have.

**Describe the solution you'd like**
After the policy_array cycles to deploy all forced / base apps to the mac, I'd like a optional second pane that the end user continues into that will allow them to select a list of apps, perhaps a checkbox or whatever may work better in this instance, with app icons, descriptions to each app in its selection phase (I am imaginging a table view, 2 across by however many long). The user will either have the option to skip this optional pane entirely by clicking continue when no selection is made, or "install" when a selection of these apps are made. The same workflow will follow from policy_array to show installation progress etc and then will move to the completed tab where a user's end action is initiated.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by dan-snelson** [comment]
Thanks, @GadgetGeekNI. 

(It'd be nice to have Issues [108](https://github.com/bartreardon/swiftDialog/issues/108) and [191](https://github.com/bartreardon/swiftDialog/issues/191) resolved.)

**Comment by dan-snelson** [comment]
@GadgetGeekNI:

While not _precisely_ what you requested, please advise if the new "Configurations" in SYM `1.8.0-rc1` will meet your needs:
- [Required](https://github.com/dan-snelson/dialog-scripts/blob/development/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L556-L660)
- [Recommended](https://github.com/dan-snelson/dialog-scripts/blob/development/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L662-L777)
- [Complete](https://github.com/dan-snelson/dialog-scripts/blob/development/Setup%20Your%20Mac/Setup-Your-Mac-via-Dialog.bash#L779-L949)

**Comment by dan-snelson** [comment]
@GadgetGeekNI:

This Feature Request is _sort of_ addressed in `1.8.0`.

(I'm working on moving SYM to its own repo and will most likely lose this issue; if **Configurations** in `1.8.0` don't meet your needs, please open a fresh issue once the repo move is complete.)

**Comment by mueller-mario** [comment]
Hi Dan,

I think that's a good approach with the 3 variants for installation.
Regards, Mario.
________________________________
Von: Dan K. Snelson ***@***.***>
Gesendet: Sonntag, 5. März 2023 01:14
An: dan-snelson/dialog-scripts ***@***.***>
Cc: Mario Mueller ***@***.***>; Manual ***@***.***>
Betreff: Re: [dan-snelson/dialog-scripts] Optional Apps Pane after inital policy_array (Issue #37)


@GadgetGeekNI<https://github.com/GadgetGeekNI>:

This Feature Request is sort of addressed in 1.8.0.

(I'm working on moving SYM to its own repo and will most likely lose this issue; if Configurations in 1.8.0 don't meet your needs, please open a fresh issue once the repo move is complete.)
